### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by cdeil** [hplgit/web4sciapps#2]
I just found this tutorial by Googling for how to send matplotlib plots to the browser ... it's exactly what I want to learn ... thank you!!!

I just started reading and saw that you're using the print statement in some examples:
http://hplgit.github.io/web4sciapps/doc/pub/._web4sa_solarized002.html
https://github.com/hplgit/web4sciapps/search?utf8=%E2%9C%93&q=print

Probably it's not too hard to make everything work with Python 3 as well?

**Comment by hplgit** [comment]
No, it should be straightforward, but it would be an idea to use python-future to create a common code basis for Python 2 and 3 since many math libraries are not ported to Python 3 so the apps most likely must run under Python 2. I try to code v2.7 in a way that makes the distance to v3.4/3.5 as small as possible.
